### ISSUE TITLE
plot_models with order.term option [feature request]

### ISSUE THREAD

**Issue by albertostefanelli** [strengejacke/sjPlot#452]
By default, the function sorts the estimates with highest at the top. Would be nice to have something similar to order.term as in plot_model

**Comment by joanneleung** [comment]
Agree! Also, is there a way to add labels to group variables on the y-axis? Thanks!

**Comment by fariedd** [comment]
Plot_models is an awesome package, but there are no order.terms  so the desired data has a more presentable format.
Help please thank you
